src/components/elements/Checkbox/Readme.md

```diff
@@ -0,0 +1,128 @@
+### Usage
+
+```jsx static
+import { Checkbox } from '@codespec/react-toolset'
+```
+
+### Description
+
+`<Checkbox>` is a react component of native checkbox.
+
+### Basic Checkbox
+
+```jsx
+initialState = { value: false, primary: false };
+<div>
+  <Checkbox
+    id="checkbox_example"
+    isChecked={state.value}
+    onChange={() => setState({ value: !state.value})}>
+    Default Checkbox
+  </Checkbox>
+  <br />
+  <Checkbox
+    id="primary_checkbox_example"
+    type="primary"
+    isChecked={state.primary}
+    onChange={() => setState({ primary: !state.primary})}>
+    Primary Checkbox
+  </Checkbox>
+</div>
+```
+
+### Sizes
+
+```jsx
+initialState = {
+  small: false, medium: false, large: false, xlarge: false,
+  priamrySmall: false, primaryMedium: false, primaryLarge: false, primaryXlarge: false,
+};
+<div>
+  <div className="checkboxExampleSection">
+    <Checkbox
+      id="small_checkbox_example"
+      size="small"
+      isChecked={state.small}
+      onChange={() => setState({ small: !state.small})}
+      cls={{root: "checkboxInlineExample"}}>
```

Comment:
Q: what is `cls`?  is this for class names? can we have a more descriptive name?